src/core/store/common/tokensByAsset.selectors.ts

```diff
@@ -70,6 +70,7 @@ export const selectDepositTokenOptionsByAsset = createSelector(
             const allowancesMap = userTokensAllowancesMap[address] ?? {};
             return createToken({ tokenData, userTokenData, allowancesMap });
           });
+        subgraphTokens = sortBy(subgraphTokens, (o) => o.symbol);
```

Comment:
This is unrelated to the PR, but I noticed there was a bug from when @kibagateaux  and I reviewed my last PR to add supported oracle tokens (the bug was that the list was not appropriately sorted).

Reply:
I HATE mutating variables. Either chain sortBy during initial array construction or create a new variable.

Reply:
This is causing rendering errors and is not related to the Portfolio page query issue, should this be in it's own PR?

Reply:
@Seroxdesign How is this causing rendering errors?

@kibagateaux I created a new variable to remove the variable mutation. Thanks for the feedback - I'll adopt the practice of NOT mutating variables moving forward.

Reply:
I believe it was the missing API key that would cause the app to crash, I'm going to test this out again and let you know if the issue still occurs.